ohttp/src/lib.rs

```diff
@@ -116,6 +116,23 @@ impl KeyConfig {
         })
     }
 
+    /// Derive a configuration for the server side from input keying material
```

Comment:
My first thought was that this was supposed to follow: https://www.ietf.org/archive/id/draft-irtf-cfrg-hpke-12.html#name-derivekeypair  Is that right?  Want to cite that?